lib/awesome_print/custom_defaults.rb

```diff
@@ -0,0 +1,46 @@
+module AwesomePrint
+  class << self
+    attr_accessor :defaults, :force_colors
+
+    # Class accessor to force colorized output (ex. forked subprocess where TERM
+    # might be dumb).
+    #------------------------------------------------------------------------------
+    def force_colors!(value = true)
+      @force_colors = value
+    end
+
+    def console?
+      !!(defined?(IRB) || defined?(Pry))
+    end
+
+    def rails_console?
+      console? && !!(defined?(Rails::Console) || ENV["RAILS_ENV"])
+    end
+
+    def diet_rb
+      IRB.formatter = Class.new(IRB::Formatter) do
+        def inspect_object(object)
+          object.ai
+        end
+      end.new
+    end
+
+    def usual_rb
+      IRB::Irb.class_eval do
+        def output_value
+          ap @context.last_value
+        end
```

Comment:
I think it's just missing

```ruby
rescue NoMethodError        
  puts "(Object doesn't support #ai)"
```

Which was recently added by a pull request.

#212